### PR TITLE
test: Add tests for View report flow [2]

### PR DESCRIPTION
**Pull request by bistaastha**

### CODE REVIEW

**Review by Chethan-Fyle — APPROVED** [review]

File: src/app/fyle/my-view-report/my-view-report.page.spec.ts [hunk 1]
```diff
@@ -233,7 +239,7 @@ describe('MyViewReportPage', () => {
     orgSettingsService = TestBed.inject(OrgSettingsService) as jasmine.SpyObj<OrgSettingsService>;
     spenderReportsService = TestBed.inject(SpenderReportsService) as jasmine.SpyObj<SpenderReportsService>;
 
-    component.erpt$ = of(newReportParam);
+    component.erpt$ = of(platformReportData);
```
Comment (mvaishnavi) [inline src/app/fyle/my-view-report/my-view-report.page.spec.ts:242]: reports

File: src/app/fyle/my-view-report/my-view-report.page.spec.ts [hunk 2]
```diff
@@ -488,7 +494,7 @@ describe('MyViewReportPage', () => {
   });
 
   it('updateReportName(): should update report name', () => {
-    const mockErpt = cloneDeep(newReportParam);
+    const mockErpt = cloneDeep(platformReportData);
```
Comment (mvaishnavi) [inline src/app/fyle/my-view-report/my-view-report.page.spec.ts:497]: report

File: src/app/fyle/my-view-report/my-view-report.page.spec.ts [hunk 3]
```diff
@@ -488,7 +494,7 @@ describe('MyViewReportPage', () => {
...
     component.erpt$ = of(mockErpt);
```
Comment (mvaishnavi) [inline src/app/fyle/my-view-report/my-view-report.page.spec.ts:498]: report

File: src/app/fyle/my-view-report/my-view-report.page.spec.ts [hunk 4]
```diff
@@ -501,7 +507,7 @@ describe('MyViewReportPage', () => {
 
   describe('editReportName(): ', () => {
     it('should edit report name', fakeAsync(() => {
-      component.erpt$ = of(cloneDeep({ ...expectedAllReports[0], rp_state: 'DRAFT' }));
+      component.erpt$ = of(cloneDeep({ ...platformReportData, state: 'DRAFT' }));
```
Comment (mvaishnavi) [inline src/app/fyle/my-view-report/my-view-report.page.spec.ts:510]: report

File: src/app/fyle/my-view-report/my-view-report.page.spec.ts [hunk 5]
```diff
@@ -554,19 +560,19 @@ describe('MyViewReportPage', () => {
   });
 
   it('deleteReport(): should delete report', () => {
-    component.erpt$ = of(expectedAllReports[0]);
+    component.erpt$ = of(expectedReportsSinglePage[0]);
```
Comment (mvaishnavi) [inline src/app/fyle/my-view-report/my-view-report.page.spec.ts:563]: report

File: src/app/fyle/view-team-report/view-team-report.page.html [hunk 6]
```diff
@@ -66,7 +66,7 @@
               <div class="view-reports--employee-name">
                 <div>
                   <span class="view-reports--employee-name__text">Employee : </span>
-                  <span class="view-reports--employee-name__name">{{erpt.us_full_name | ellipsis: 30}}</span>
+                  <span class="view-reports--employee-name__name">{{erpt.employee.user.full_name | ellipsis: 30}}</span>
```
Comment (mvaishnavi) [inline src/app/fyle/view-team-report/view-team-report.page.html:69]: report.

**Review by mvaishnavi — CHANGES_REQUESTED** [review]
will review this again after the erpt is removed.

**Review by mvaishnavi — APPROVED** [review]

**Pull request merged**